action.yaml

```diff
@@ -35,22 +35,22 @@ inputs:
   wait-for-completion:
     description: 'Block until the triggered workflow has finished'
     required: false
-    default: true
```

Comment:
Same here, why ?

Reply:
And this: https://github.com/aurelien-baudet/workflow-dispatch/blob/00dba38060567621bfc0f88e7fc19a4932f4312f/src/utils.ts#L49